tests/test_phshift2007.py

```
@@ -1,7 +1,29 @@
 
 def test_e2e():
+    if os.environ.get("PHASESHIFTS_SKIP_PHSHIFT2007_DOWNLOAD"):
+        pytest.skip("PHASESHIFTS_SKIP_PHSHIFT2007_DOWNLOAD is set")
+    if not find_executable("gfortran"):
+        pytest.skip("gfortran is not available")
     with tempfile.TemporaryDirectory() as directory:
-        phaseshifts.phshift2007.do_action("all", directory)
+        try:
+            phaseshifts.phshift2007.do_action("all", directory)
+        except (URLError, ssl.SSLError) as exc:
+            pytest.skip("phshift2007 download unavailable: %s" % exc)
```

Ruff (0.14.10)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)